crates/astria-sequencer/src/app/mod.rs

```diff
@@ -458,6 +482,16 @@ impl App {
         );
 
         self.executed_proposal_hash = process_proposal.hash;
+        self.post_execute_transactions(
```

Comment:
Why not pass this the entire `process_proposal`?

Reply:
this is also called in `finalize_block` potentially, so it might not have `process_proposal` available